internal/dao/pgx/reservation_pgx.go

```diff
@@ -210,6 +210,19 @@ func (x *reservationDao) UpdateStatus(ctx context.Context, id int64, status stri
 	return nil
 }
 
+func (x *reservationDao) UnscopedUpdateAWSDetail(ctx context.Context, id int64, awsDetail *models.AWSDetail) error {
+	query := `UPDATE aws_reservation_details SET detail = $2 WHERE reservation_id = $1`
+
+	tag, err := db.Pool.Exec(ctx, query, id, awsDetail)
+	if err != nil {
+		return fmt.Errorf("pgx error: %w", err)
+	}
+	if tag.RowsAffected() != 1 {
+		return fmt.Errorf("expected 1 row: %w", dao.ErrAffectedMismatch)
+	}
+	return nil
+}
```

Comment:
We should be generally trying to remove all "unscoped" DAO functions in the future, it's a bit more work to join tables to get an account, but it is dramatically safer if we misuse this in the future. I am okay to merge this in, unless you want to do an extra mile.

Reply:
Well I don't want to. At least not without discussion, as complicating the queries might have some performance impact, I'd love to discuss if we don't want to do it in two queries. Which makes it a problem on its own IMHO and would slow down this PR which is already quite big and will be hard to get in :)

Reply:
You want to discuss SQL JOIN? Okay, but prepare good arguments for "joins are slow" in this case. My arguments will be "it's security, let's be extra careful". We already do this in several functions, I remember writing this myself.

Reply:
Given the function prerix, I do not want to block this PR on this, but generally I would like us not to have any Unscoped functions, I do not see big reason not to join accounts table on a query that returns a _single row_.